apps/src/code-studio/components/AdvancedShareOptions.jsx

```diff
@@ -56,18 +56,19 @@ const AdvancedShareOptions = Radium(React.createClass({
       iframeHeight: PropTypes.number.isRequired,
       iframeWidth: PropTypes.number.isRequired,
     }).isRequired,
-  },
+  };
 
-  getInitialState() {
-    return {
-      selectedOption: (this.props.onClickExport && 'export') || 'embed',
+  constructor(props) {
+    super(props);
+    this.state = {
+      selectedOption: (props.onClickExport && 'export') || 'embed',
```

Comment:
This seems better expressed as `props.onClickExport ? 'export' : 'embed'`, but I guess that's out of scope for this change.